implementation.js

```diff
@@ -157,6 +157,8 @@ module.exports = function from(arrayLike) {
 	if (arrayFromIterable) {
 		items = arrayFromIterable;
 		len = arrayFromIterable.length;
+	} else if (typeof arrayLike === 'string') {
+		return items.match(/[\uD800-\uDBFF][\uDC00-\uDFFF]?|[^\uD800-\uDFFF]|./g) || [];
```

Comment:
this should not happen unless it's in an environment without `Symbol.iterator` - how can we ensure that? If the environment has that, we must look up the `Symbol.iterator` function off of the string, and obey it.